packages/core/core/src/Parcel.js

```diff
@@ -201,10 +201,12 @@ export default class Parcel {
       this.#assetGraphBuilder.writeToCache(),
       this.#runtimesAssetGraphBuilder.writeToCache(),
     ]);
+    this.#farm.clearConfigCache();
   }
 
   async _startNextBuild() {
     this.#watchAbortController = new AbortController();
+    this.#farm.clearConfigCache();
```

Comment:
```suggestion
    await this.#farm.callAllWorkers('clearConfigCache', []);
```

In worker.js, you'll need to add this:

```js
export function clearConfigCache() {
  loadConfig.clearCache();
}
```

Each worker has its own individual cache, so we need to call all of them to clear them. The `callAllWorkers` function will take care of calling the `clearConfigCache` function in worker.js inside each worker.